presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -169,6 +169,8 @@ Property Name                                      Description
                                                    See :ref:`hive-file-based-authorization` for details.
 
 ``hive.non-managed-table-writes-enabled``          Enable writes to non-managed (external) Hive tables.         ``false``
+
+``hive.create-non-managed-table-enabled``          Enable creating non-managed (external) Hive tables.          ``true``
```

Comment:
I tested this case. S3 disallows creating another table, say named `location` if there is an existing directory `s3://some/location` (You can still create a table with a different name, but then such tables won't be auto-populated with the data under `s3://some/location`). Also, as I understand, the only way to create an external table is when you specify `WITH (external_location = ...)` clause, which is what this path is about.